webdriver/tests/bidi/browsing_context/set_viewport/invalid.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+import webdriver.bidi.error as error
+
+
+@pytest.mark.parametrize("value", [None, False, 42, {}, []])
+@pytest.mark.asyncio
+async def test_params_context_invalid_context(bidi_session, value):
```

Comment:
Can you please use the same naming convention as what we have for all the other BiDi API's?
```suggestion
async def test_params_context_invalid_type(bidi_session, value):
```

We also need a test for an invalid value.